c2corg_ui/templates/helpers/view.html

```diff
@@ -120,20 +120,17 @@ <h3 class="text-center"><span translate>Merge documents</span> (<span x-translat
   </div>
 </%def>
 
-<%def name="show_delete_document_dialog()">\
+<%def name="show_delete_document_dialog(module)">\
+  <%
+      # get the module without the 's' at the end ('routes' -> 'route')
+      # to reuse existing translation strings
+      document_type = module[:-1]
```

Comment:
Not sure it is useful to add put an ending ``s`` to remove it afterward.
Should be simpler to pass the ``document_type`` string to the ``show_delete_document_dialog``